samples/src/main/java/org/corfudb/samples/WriteWriteTXs.java

```diff
@@ -115,7 +117,9 @@ void action() {
             try {
                 th.join(THREAD_TIMEOUT);
             } catch (InterruptedException ie) {
+                Thread.currentThread().interrupt();
                 System.out.println("Thread timeout!");
+                throw new RuntimeException(ie);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)

Reply:
done,